app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
Similar to the sylesheets, no reason to create this file if you don't have any helpers.